Neighborlists and Dangerous Builds.ipynb

```diff
@@ -8,18 +8,18 @@
     "\n",
```

Comment:
<blockquote>may <strong>have  be missed</strong>, possibly  leading to incorrect behavior. </blockquote><p>may have been</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PTC-CMC/cybercamp/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>